Content.Server/GameTicking/GameTicker.Player.cs

```diff
@@ -81,6 +82,21 @@ private async void PlayerStatusChanged(object? sender, SessionStatusEventArgs ar
                 {
                     _userDb.ClientConnected(session);
 
+                    if (mind == null)
+                    {
+                        // try to load persistence saved mind for this player when joining
+                        var query = EntityManager.AllEntityQueryEnumerator<MindComponent>();
+                        while (query.MoveNext(out var mindComp))
+                        {
+                            if (mindComp.UserId == session.UserId)
+                            {
+                                // found a mind, later it will take over its entity
+                                mind = mindComp;
+                                break;
+                            }
+                        }
+                    }
+
```

Comment:
As far as I know, this block of code is unnecessary now (maybe it wasn't when the PR was first created).

If this bit of code finds a mind, then `SharedMindSystem.OnMindStartup` should've added it to the user-mind dictionary so that the `_mind.TryGetMind()` call at the beginning of the function should already have found the mind.

If it doesn't work, there is probably some other bug elsewhere that needs fixing instead of doing it like this?